guide/preferences.md

```diff
@@ -32,25 +32,18 @@ If available, a dialog window will inform you about it.
 ### OpenGL
 
 The **OpenGL** tab contains preferences about setting the 3D rendering abilities.
-The initial parameters of these settings may vary from one computer to another depending on the GPU OpenGL abilities.
+The default parameters of these settings may vary from one computer to another depending on the system's hardware & OpenGL cabilities.
 
-- The **RTT preferred mode** option allows you to modify the method used to create the Camera device images.
-The methods are sorted from the most efficient one to the least efficient one.
+- The **Main 3D view anti-aliasing** option allows you to enable Anti-Aliasing, specifically [SMAA 1x](http://www.iryoku.com/smaa/) on the 3D scene in Webots.
+This option can lead to marginally reduced performance, but it improves graphical fidelity somewhat.
 
-- The **Main 3D view anti-aliasing** option allows you to enable Multisample Anti-Aliasing on the 3D scene in Webots.
-This option can lead to reduced performance, but it improves graphical fidelity significantly.
-It is disabled by default on systems that do not meet our minimum requirements.
-Note that this option does not apply to any Camera rendering, this is managed by the `Disable camera anti-aliasing` setting in the same tab of the preferences dialog.
+> **Note** This option does not apply to any Camera rendering, this is managed by the `Disable camera anti-aliasing` setting in the same tab of the preferences dialog.
```

Comment:
I think that now that we have the possibility to do cross-book reference it is good to add links to the device in the reference manual when editing a part with a device.